### DISCUSSION TITLE
Is toonsquid good for longer projects?

### DISCUSSION THREAD

**Discussion by MustardMonday** [keiwando/toonsquid#359] [Q&A]
I'm still really new and learning when it comes to animation, but I find toonsquid is a perfect mix of having multiple advanced tools, that aren't too hard to use. I was thinking of starting a longer project and was debating if toonsquid would be a good platform for let's say a 10-30 minute animation? Any advice or suggestions?

**Comment by keiwando** [comment]
Hi! ToonSquid does not have a fixed length limit:
https://toonsquid.com/handbook/guides/faq/#length-limit

You might still want to split such a long animation into multiple scenes for organizational purposes.

**Reply by MustardMonday** [reply]
Thank you! I'm super excited to continue working with toonsquid!
